src/components/timeline/LayerLine.svelte

```diff
@@ -62,7 +66,7 @@
   $: onContextMenu(contextmenu);
   $: onMousemove(mousemove);
   $: onMouseout(mouseout);
-  $: points = resourcesToLinePoints(resources, pointRadius);
+  $: resources && processResourcesToLinePoints(resources);
```

Comment:
should this be `resources.length > 0` or is there actually a case where `resources` is null/undefined?

Reply:
I think either could work actually, all I'm really going for here is to kick off `processResourcesToLinePoints` any time the reference to resources changes.

Reply:
Out of curiosity, would just `processResourcesToLinePoints(resources);` not be enough to trigger a rerun if `resources` changes? Not saying that you should change it to this, just wondering.

Reply:
Think it would trigger it?